.dccache

```diff
@@ -0,0 +1 @@
+{"/Users/artjom.kurapov/git/graphql-query-cost/lib/calculateCost.js":[733,1653467093473.2205,"3d4161e2d836f5957e4412b98c2a3334ae0cfa0a38b03ae41cec15324ce12c08"],"/Users/artjom.kurapov/git/graphql-query-cost/lib/calculateCost.recursion.test.js":[9168,1653467093473.9697,"dcd9cc9e4b890c399a41979db6b844981d51ac058a358024bd27b75b07feb795"],"/Users/artjom.kurapov/git/graphql-query-cost/lib/calculateCost.test.js":[13669,1653467093475.097,"cf082de4c9b580d75eb42a043fe3ff058f26b717f75ab9390ff47526e8fdb3bc"],"/Users/artjom.kurapov/git/graphql-query-cost/lib/costDirective.js":[182,1647509705696.3625,"76b1460499adb49ba06ee2da811aeaf06b340e71505db33efab58eae09eea46c"],"/Users/artjom.kurapov/git/graphql-query-cost/lib/costExtractor.js":[3248,1647509705696.5283,"cb5a6a9f3adee3030f0719bd4f9b4eb58602ed92a2e1df89e228889c61d7b83d"],"/Users/artjom.kurapov/git/graphql-query-cost/lib/costExtractor.test.js":[3054,1647509705696.6973,"76f68cb76a83e4117b737ec8633b152f72c7e51738d32afb5f216a8e7d113161"],"/Users/artjom.kurapov/git/graphql-query-cost/lib/costVisitor.js":[8956,1653467093476.0002,"b8b2fc8bcd2bb00bca75b4f2f21e4fb6afaaabd312a18353b8fcbef69a42bd98"],"/Users/artjom.kurapov/git/graphql-query-cost/lib/error.js":[265,1653467093476.4268,"727c2d4be7d77a1069adee3687b1a03cfc2dc0ccae7be48a4ed8d0d5a6daf3d4"],"/Users/artjom.kurapov/git/graphql-query-cost/lib/index.js":[175,1647509705697.0444,"6172a50356a2def12530fd7afd775d019cbd0e27b4abc7c4e50fc4b1362058c9"],"/Users/artjom.kurapov/git/graphql-query-cost/examples/express-apollo/index.js":[1770,1647509705694.9495,"c42fba624e790d2b9b3f1715904edb1dfaaa32f9edfe6459598b2b6e7366100b"],"/Users/artjom.kurapov/git/graphql-query-cost/test/unit/jest.config.js":[179,1647509705699.3884,"9c8110cf7eb39744ee6e01f21762c4bcce9a16102ef51d8abd025d79f9f9c8f7"],"/Users/artjom.kurapov/git/graphql-query-cost/test/unit/jest.coverage.js":[404,1647509705699.5435,"cc261d1872b0cb75c79755e96d71d7704cafeca86f7c47b2bba970a9afd12833"]}
```

Comment:
I think this file should be .gitignore-d as it is user specific?

Reply:
thanks, removed